client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -40,9 +41,10 @@ const StatusBox: React.FC<IStatusBoxProps> = ({
   buttonLabel,
   onButtonClick,
   children,
+  success,
 }: IStatusBoxProps) => {
   return (
-    <Wrapper>
+    <Wrapper intent={success ? 'success' : 'none'} icon={null}>
```

Comment:
i think you should remove the status bar changes and just go with the upload success message, since it's still confusing in the way we previously discussed

Reply:
Fair enough. :slightly_smiling_face:

Reply:
I did leave the 'successfully' addition in the header, though it's not green anymore. I think that helps clarify things a bit more without being confusing.